opensearch-operator/opensearch-gateway/responses/ClusterHealthResponse.go

```diff
@@ -0,0 +1,10 @@
+package responses
+
+type ClusterHealthResponse struct {
+	Status             string  `json:"status,omitempty"`
+	ActiveShards       int     `json:"active_shards,omitempty"`
+	RelocatingShards   int     `json:"relocating_shards,omitempty"`
+	InitializingShards int     `json:"initializing_shards,omitempty"`
+	UnassignedShards   int     `json:"unassigned_shards,omitempty"`
+	PercentActive      float32 `json:"active_shards_percent_as_number,omitempty"`
+}
```

Comment:
Maybe we need to add some more nodes (data/masters) health status ?

Reply:
Sure, although I think those would need to be queried separately?  The cluster health endpoint only gives information about the shard health.

Reply:
you right, we will get to it